eclide/EclDlgBuilder.cpp

```diff
@@ -40,6 +40,13 @@ void CBuilderDlg::GetTitle(CString & title)
     title = (m_view.IsDirty() ? _T("*") : _T("")) + name;
 }
 
+void CBuilderDlg::PathToClipboard() {
+    std::_tstring path = m_attribute->GetQualifiedLabel(false, true);
```

Comment:
I thought the m_attribute had a getPath function (when its an IFileAttribute)?

Reply:
@GordonSmith getPath gets the entire path including the drive and repository location. The request was for the relative path.

Reply:
You can add Relative Path to the DiskAttribute class?  (would save duplicating this code).

Reply:
@GordonSmith can do. Good idea.